### PR TITLE
{bio}[system/system] FragPipe v20.0

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/b929f95f1b7805bcc3f6d0f7f8e785dc for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/f/FragPipe/FragPipe-20.0-Java-11.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+checksums = ['0d77db85ae26c190a915823e1c68238f3a19533810734b008564b2a8d85d1bd6']
+
+dependencies = [('Java', '11')]
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'lib/%(namelower)s-%(version)s.jar'],
+    'dirs': [],
+}
+
+sanity_check_commands = [("%(namelower)s -h | grep -q v%(version)s")]
```
Comment (boegel) [inline easybuild/easyconfigs/f/FragPipe/FragPipe-20.0-Java-11.eb:24]:
```suggestion
sanity_check_commands = ["%(namelower)s -h | grep -q v%(version)s"]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3143.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/2c0f3667c16a3aba892aa30a42a3c6df for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**